src/lib/snack-bar/snack-bar-ref.ts

```diff
@@ -55,19 +55,19 @@ export class MdSnackBarRef<T> {
     clearTimeout(this._durationTimeoutId);
   }
 
-  /** Dismisses the snack bar after some duration */
-  _dismissAfter(duration: number): void {
-    this._durationTimeoutId = setTimeout(() => this.dismiss(), duration);
-  }
-
   /** Marks the snackbar action clicked. */
-  _action(): void {
+  closeWithAction(): void {
```

Comment:
Since this is a public API now, could you add a unit test that is based around calling this? The current unit test only deals with clicking the button directly.